services/web-server/src/login/strategies/github.js

```diff
@@ -22,12 +24,31 @@ module.exports = class Github {
 
     this.rootUrl = cfg.taskcluster.rootUrl;
     this.identityProviderId = 'github';
+    this.githubClient = new GithubClient();
   }
 
-  async getUser({ userId }) {
+  async getUser({ username, userId }) {
     const user = new User();
 
-    user.identity = `${this.identityProviderId}/${encode(userId)}`;
+    if (!username) {
```

Comment:
Is this just in case something else is calling getUser?

Reply:
Can't think of why this was needed. Let's remove it.